src/test/ts/commands/StatusTest.ts

```diff
@@ -87,24 +105,7 @@ describe('Status', () => {
       },
       versionString: '1.98.7'
     });
-  });
-
-  it('shows status for releaseReady of "old" release', async () => {
-    const { dir, git } = await newGit();
-    await branchWithPj({ dir, git }, '41.98.7', 'release/41.98');
-    await branchWithPj({ dir, git }, '1.100.1', 'release/1.100');
+  }).timeout(20000);
 
-    await check(dir, {
-      branchState: 'releaseReady',
-      isLatestReleaseBranch: false,
-      currentBranch: 'release/1.100',
-      branchType: 'release',
-      version: {
-        major: 1,
-        minor: 100,
-        patch: 1
-      },
-      versionString: '1.100.1'
-    });
-  });
+  // TODO: Add test case where isLatest returns false. Will need to publish, or mock out listing tags.
```

Comment:
Is this a "before it gets merged" TODO or a "Jira this up for later" TODO

Reply:
Jira.
Done.